src/kolibri_android/main_activity/activity.py

```diff
@@ -186,13 +186,15 @@ def replace_url(self, url):
         replace_url_in_webview(url)
 
     def start_kolibri(self):
+        from kolibri.core.device.models import DeviceAppKey
+
         # TODO: Wait until external storage is available
         #       <https://phabricator.endlessm.com/T33974>
 
         init_kolibri(debug=True)
 
         self._kolibri_bus = _build_kolibri_process_bus(self)
-        app_key = self._kolibri_bus.get_app_key()
+        app_key = DeviceAppKey.get_app_key()
```

Comment:
This shouldn't be necessary. [`KolibriAppProcessBus.get_app_key`](https://github.com/endlessm/kolibri-installer-android/blob/develop/src/kolibri_android/main_activity/kolibri_bus.py#L28-L29) is our own addition ;) It's like that to keep imports from `kolibri` in one place.